.gcloudignore

```diff
@@ -0,0 +1,3 @@
+#!include:.gitignore
```

Comment:
Won't this prevent `dist` and `build` from being included? would we ever need those to be included?

Reply:
Yep, that's intentional so only source files are uploaded for Cloud Build. Cloud Build will install requirements when creating the docker container. Later, at runtime, Dataflow runs `setup.py` when the SDK container launches to pull in any additional dependencies.

Reply:
Ta. Good to know.